src/helper/git.ts

```diff
@@ -1,35 +1,44 @@
 // https://logseq.github.io/plugins/interfaces/IAppProxy.html#execGitCommand
-import type { IGitResult } from "@logseq/libs/dist/LSPlugin.user"
+import type { IGitResult } from "@logseq/libs/dist/LSPlugin.user";
+//import path from "path";
 
-let _inProgress: Promise<IGitResult> | undefined = undefined
+let _inProgress: Promise<IGitResult> | undefined = undefined;
 
-export const execGitCommand = async (args: string[]) : Promise<IGitResult> => {
-  if (_inProgress) await _inProgress
+export const execGitCommand = async (args: string[]): Promise<IGitResult> => {
+  if (_inProgress) await _inProgress;
 
-  let res
+  let res;
   try {
-    const currentGitFolder = (await logseq.App.getCurrentGraph())?.path
-    const runArgs = currentGitFolder ? ['-C', currentGitFolder, ...args] : args
-    _inProgress = logseq.Git.execCommand(runArgs)
-    res = await _inProgress
+    const currentGitFolder = (await logseq.App.getCurrentGraph())?.path;
+    const relCurrGitFolder = currentGitFolder
```

Comment:
This is essentially the only change other than the settings addition in `src/helper/constants.ts`

I apologise for the linter, didn't think to check the diffs on the fly before sending this PR.

Again, I'd be happy to clean it up if this seems worth a merge.